main/templates/base/form_styling.html

```diff
@@ -65,27 +84,53 @@
     });
 }
 
+var postrequests = new Map(); //var to denote if a request is in progress
```

Comment:
Oh crap I didn't realize that that bug was already in acceptation. I consider that blocking so yeah, I will look into moving that bugfix up.

Reply:
I rebased your previously merged bugfix branch onto acceptation to (hopefully) get everything on the same timeline. Because I accidentally changed your branch in place, you might notice some weirdness if you ever happen to pull from `fix/relation_requires_bug` again. Next time I'll copy the branch first!